### DISCUSSION TITLE
BitOps Community Meeting (12 Oct 2022): v2.1.0 Release, Roadmap

### DISCUSSION THREAD

**Discussion by arm4b** [bitovi/bitops#319] [Announcements]
We invite everyone (developers, adopters, or just curious people) to join the bi-weekly BitOps Community `1 hour` Meeting on Wednesday, `12 Oct 2022, 12:00 PM US East` to discuss the ongoing project developments, updates, and plans, share use cases and stories or ask/answer questions about BitOps tool.

See https://github.com/bitovi/bitops/discussions/288 for more info on how to join.

### Follow-ups from the previous #306 Meeting
- BitOps landing page (@mickmcgrath13 @armab )
  - BitOps 1-3min intro Video (@PhillypHenning)
- Update the BitOps HeyEmoji blog to v2 (@armab)
- Trying Twitch: BitOps HeyEmoji demo (@dylan-bitovi, @ccapell, @PhillypHenning)

### v2.1.0 Release
- https://github.com/bitovi/bitops/issues/307
- https://github.com/bitops-plugins/terraform/pull/7
- https://github.com/bitops-plugins/helm/pull/4
- https://github.com/bitops-plugins/kubectl/pull/2
- https://github.com/bitops-plugins/ansible/pull/2
- https://github.com/bitovi/bitops/pull/314
- Milestone Tracker: https://github.com/bitovi/bitops/milestone/7

### Roadmap https://github.com/bitovi/bitops/discussions/245 & Development Discussions
* Create a Roadmap page (docs)?
* Create a Roadmap project: https://github.com/bitovi/bitops/projects/6
* Plugins Versioning:
  * https://github.com/bitovi/bitops/discussions/206
  * https://github.com/bitovi/bitops/issues/315
  * https://github.com/bitovi/bitops/issues/316
* [Security: BitOps Plugin support for private repos #60](https://github.com/bitovi/bitops/issues/60)
* [Security: Parse passwords / credentials out of logging output #208](https://github.com/bitovi/bitops/issues/208)
* Known Bugs:
  * https://github.com/bitovi/bitops/issues?q=is%3Aissue+is%3Aopen+label%3A%22bug+🪲%22
* [Project Quality](https://github.com/bitovi/bitops/projects/4) as KPI
  * [Documentation](https://github.com/bitovi/bitops/issues?q=is%3Aissue+is%3Aopen+label%3A%22documentation+📗%22):
    * #282
    * https://github.com/bitovi/bitops/issues/302
    * https://github.com/bitovi/bitops/issues/303
    * https://github.com/bitovi/bitops/issues/143
    * https://github.com/bitovi/bitops/issues/280 (@ccapell?)
    * Cleanup docs from `TODOs`, remove unimplemented features
  * Testing
    * https://github.com/bitovi/bitops/issues/286
    * https://github.com/bitovi/bitops/issues/271
    * https://github.com/bitovi/bitops/issues/287
  * CI/CD
    * https://github.com/bitovi/bitops/issues/293
    * https://github.com/bitovi/bitops/issues/224
  * Config Schema Issues:
    * https://github.com/bitovi/bitops/labels/schema
  * Stability
  * Feature consistency
* https://github.com/bitovi/bitops/issues/173
* https://github.com/bitovi/bitops/issues/318
* Ops Repo Composability (needs a dedicated issue w description)
* [Default folder support](https://github.com/bitovi/bitops/issues/177) follow-up

**Comment by mickmcgrath13** [comment]
Call notes
# General
- Reviewed the landing [page revamp draft](https://www.bitovi.com/bitops-0?hs_preview=CSqNAdCv-87597635370)
  - BitOps video to be completed recording today (@PhillypHenning)
- [HeyEmoji blog](https://www.bitovi.com/blog/heyemoji-slack-app-aws-terraform-ansible) update for V2
  - some inconsistencies found already
  - issues have been creating

# Twitch
  - HeyEmoji focused - ease of deploy and get the value out of it. bitops in the background
  - stackstorm on EKS deployment
  - update an ops repo from v1 to v2
  - repo cleanup
    - taking existing terraform and drop it into bitops (how to reorganize bad IaC)
# 2.1 release
- Milestone: https://github.com/bitovi/bitops/milestone/7
- migrated plugins to use latest version of tools
- they should be able to pin somehow, still
- https://github.com/bitovi/bitops/issues/317
- https://github.com/bitovi/bitops/issues/315
- https://github.com/bitovi/bitops/issues/316
- Aim to release this week
- @PhillypHenning to wrap up this today: https://github.com/bitovi/bitops/pull/314
- twitter and linkedin updates once we release

# Roadmap discussion
review roadmap items in initial post of this discussion and move items to https://github.com/bitovi/bitops/projects/6

# to follow up internally
- Landing Page
  - with survey for feedback
- Marketing
